libparsec/crates/types/src/token.rs

```diff
@@ -0,0 +1,152 @@
+// Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
+
+/// The size of the generated tokens.
+const TOKEN_SIZE: usize = 16;
+/// The minimal size of the generated tokens.
+/// This is used to ensure that the tokens are not too small during deserialization.
+const TOKEN_MINIMAL_SIZE: usize = 16;
```

Comment:
Do we need this ?
I would say the token must be of a given size, so if the input data are longer than `TOKEN_SIZE` we just consider it as invalid

Reply:
The use case was if we increase the `TOKEN_SIZE` later down the road, the previous generated token are still valid if they're `>= TOKEN_MINIMAL_SIZE`.

If it's OK to render the previous tokens invalid when we change `TOKEN_SIZE` then we don't need that.

Reply:
I've changed the code to not use `TOKEN_MINIMAL_SIZE`, now the code will return `InvalidSize` if the provided hex string isn't `TOKEN_SIZE`.